cmd/neofs-adm/internal/modules/morph/container.go

```diff
@@ -373,8 +374,7 @@ func getCIDFilterFunc(cmd *cobra.Command) (func([]byte) bool, error) {
 	}
 	sort.Strings(rawIDs)
 	return func(rawID []byte) bool {
-		var v [32]byte
-		copy(v[:], rawID)
+		v := [sha256.Size]byte(rawID)
```

Comment:
overall the same review as in SDK: should be OK in most cases but once something bad happens, users see panic and we open another PR that checks len. can we just do it here now?